st2tests/st2tests/config.py

```diff
@@ -283,6 +283,9 @@ def _register_ssh_runner_opts():
 
 def _register_scheduler_opts():
     scheduler_opts = [
+        cfg.FloatOpt(
+            'execution_scheduling_timeout_threshold_min', default=5,
```

Comment:
The test config should also default to 1 min as well to match the app config.  Sorry if this wasn't clear from my last review comment.